src/app/core/tools/handlers/SelectPathHandler.ts

```diff
@@ -2,33 +2,36 @@ import {LEFT_MOUSE_BUTTON} from "core/utils/Constants";
 
 import {Event} from "core/utils/Events";
 import {CircuitInfo} from "core/utils/CircuitInfo";
-import {GetPath} from "core/utils/ComponentUtils";
+import {GetComponentPath, GetWirePath} from "core/utils/ComponentUtils";
 
 import {CreateGroupSelectAction} from "core/actions/selection/SelectAction";
 
 import {EventHandler} from "../EventHandler";
 
+import {Component} from "core/models";
+import {Wire} from "core/models";
```

Comment:
nit: Combine these to `import {Component, Wire} from "core/models";` and more the import to be above the ".." import